src/com/lftechnology/java/training/alina/basic/gameleague/GameLeagueScores.java

```diff
@@ -0,0 +1,34 @@
+package com.lftechnology.java.training.alina.basic.gameleague;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Class GameLeagueScores shows the number of scores of wins, draws, defeats
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class GameLeagueScores {
+	private static final Logger LOGGER = Logger
+			.getLogger(GameLeagueScores.class.getName());
+	private static String getScores = null;
+
+	public static void main(String[] args) {
+		try {
+			String results = "Manchester United 1 Chelsea 0,"
+					+ " Arsenal 1 Manchester United 1,"
+					+ " Manchester United 3 Fulham 1,"
+					+ " Liverpool 2 Manchester United 1,"
+					+ " Swansea 2 Manchester United 4";
+			String resultSet[] = results.split(",");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)